src/ui/react/src/components/base/widget-position.js

```diff
@@ -210,15 +210,14 @@
                     top: endPoint[1] + 'px',
                     opacity: 1
                 });
+				
+				// 150ms to match transition-duration for .ae-toolbar-transition:
+				setTimeout(() => {
+					domElement.setStyles({
+						pointerEvents: '',
+					});
+				}, 150);
```

Comment:
So on the master branch we use 100ms here, not 150ms. Do you think we should change it there as well @rolandpakai? Do you remember why you picked 100ms @julien?

I'll merge this as-is in any case, but we can loop back and do the update later if you think we should.